kong/db/migrations/helpers.lua

```diff
@@ -186,7 +186,11 @@ function _M:copy_cassandra_records(source_table_def,
                             dest_column_name, source_table_def[dest_column_name])
           end
 
-          source_value = type_converter(source_value)
+          if source_value == nil then
+            source_value = cassandra.null
```

Comment:
An old problem that keeps resurfacing that I recalled: we really should use `cassandra.unset` instead, which helps with newer Cassandra versions:

http://thelastpickle.com/blog/2016/09/15/Null-bindings-on-prepared-statements-and-undesired-tombstone-creation.html

https://thibaultcha.github.io/lua-cassandra/modules/cassandra.html#type_serializers

Reply:
```suggestion
            source_value = cassandra.unset
```